Document/0x03-Overview.md

```diff
@@ -1,75 +1,86 @@
+
 # Introduction to the OWASP Mobile Security Testing Guide
 
-The OWASP Mobile Security Testing Guide (MSTG) is an extension of the OWASP Testing Project specifically focusing on the security testing of Android and iOS devices.
+The OWASP Mobile Security Testing Guide (MSTG) is an extension of the OWASP Testing Project that performs security testing of Android and iOS mobile devices.
+
+The goal of this project is to help people understand everything they need to know about testing Android and iOS applications. MSTG delivers a complete suite of test cases designed to address the [OWASP Mobile Top 10 security risks](https://www.owasp.org/index.php/Mobile_Top_10_2016-Top_10) and the requirements of the [Mobile Application Security Verification Standard (MASVS)](https://github.com/OWASP/owasp-masvs).
+
+>Do we want to give links to those 3 resources/standards mentioned above (if these links exist)? Do we want to link to the OWASP Testing Project? May also want to link the first instance of OWASP to the main site? Are we avoiding links? What are the ways/forms this book is being published?*
+
+## Why Do We Need a Mobile Application Security Testing Guide?
 
-The goal of this project is to help people understand the what, why, when, where, and how of testing applications on Android and iOS devices. The project delivers a complete suite of test cases designed to address the OWASP Mobile Top 10, the Mobile App Security Checklist and the Mobile Application Security Verification Standard (MASVS).
+New technology always introduces new security risks, and mobile computing is no exception. Security concerns for mobile apps differ from traditional desktop software in some important ways, including portability and novelty. Modern mobile operating systems are arguably more secure than traditional desktop operating systems, but problems can still appear when we don't carefully consider security during mobile app development. Data storage, inter-app communication, proper usage of cryptographic APIs, and secure network communication are only some of these considerations. 
 
-## Why Does the World Need a Mobile Application Security Testing Guide?
+>The second paragraph that used to be here is kinda repeated below in Local Data Storage, so I removed the duplicate content and just hinted at it, keeping the explanation down below. 
 
-Every new technology introduces new security risks, and mobile computing is no different. Even though modern mobile operating systems like iOS and Android are arguably more secure by design compared to traditional Desktop operating systems, there's still a lot of things that can go wrong when security is not considered during the mobile app development process. Data storage, inter-app communication, proper usage of cryptographic APIs and secure network communication are only some of the aspects that require careful consideration.
+## Key Areas in Mobile Application Security (AppSec)
```

Comment:
I think the "(AppSec)" abbreviation can be removed, especially in the section header. We don't use it in many places, and we should write it out where we do, so there is no need to define AppSec here.